cw/cw305/capture.py

```diff
@@ -200,26 +311,88 @@ def capture_aes_fvsr_key(ot):
         yield ret
 
 
+def capture_aes_mix_column(ot, ktp):
+    """A generator for capturing AES traces.
+    Fixed key, Random texts. 4 sets of traces. Mix column HD CPA Attack.
+
+    Attack implemented by ChipWhisperer:
+    Repo: https://github.com/newaetech/chipwhisperer-jupyter/blob/master/experiments/MixColumn%20Attack.ipynb
+    Reference: https://eprint.iacr.org/2019/343.pdf
+
+    See mix_columns_cpa_attack.py for attack portion.
+
+    Args:
+      ot: Initialized OpenTitan target.
+      ktp: Key and plaintext generator.
+    """
+    key, _ = ktp.next()
+    tqdm.write(f'Using key: {binascii.b2a_hex(bytes(key))}')
+
+    cipher = AES.new(bytes(key), AES.MODE_ECB)
+    while True:
+        _, text = ktp.next()
+        ret = cw.capture_trace(ot.scope, ot.target, text, key, ack=False, as_int=True)
+        if not ret:
+            raise RuntimeError('Capture failed.')
+        expected = binascii.b2a_hex(cipher.encrypt(bytes(text)))
+        got = binascii.b2a_hex(ret.textout)
+        if got != expected:
+            raise RuntimeError(f'Bad ciphertext: {got} != {expected}.')
+        yield ret
+
+
 @app_capture.command()
 def aes_random(ctx: typer.Context,
-               num_traces: int = opt_num_traces,
-               plot_traces: int = opt_plot_traces):
-    """Capture AES traces from a target that runs the `aes_serial` program."""
+        num_traces: int = opt_num_traces,
+        plot_traces: int = opt_plot_traces):
+    """Capture AES traces. Fixed key random texts."""
```

Comment:
These changes are unrelated to the batch capture work. They should go into a seperate commit. Also, it introduces again lint errors. Can you please run `flake8 capture.py --max-line-length 100` and fix all errors. Thanks!

Reply:
Oh OK.

Reply:
Thanks.